services/QuillLMS/app/controllers/cms/users_controller.rb

```diff
@@ -271,9 +272,10 @@ def complete_sales_stage
       WHERE classrooms.code = #{sanitized_class_code}))"
   end
 
-  protected def pagination_query_string
-    page = [user_query_params[:page].to_i - 1, 0].max
-    "LIMIT #{USERS_PER_PAGE} OFFSET #{USERS_PER_PAGE * page}"
+  protected def limit_query_string
+    return '' if user_query_params[:record_limit].to_s.downcase == 'all'
+
+    "LIMIT #{user_query_params[:record_limit] || USERS_PER_PAGE}"
```

Comment:
It looks like this new query segment doesn't let you fetch anything other than the first page of results, but there's still a `page` param that's passable.  Probably doesn't really matter, but might be confusing if someone comes in later trying to figure out why fetching "page 2" just provides the data from page 1.